patroni/__main__.py

```diff
@@ -81,12 +82,32 @@ def ensure_unique_name(self) -> None:
             return
 
     def get_tags(self) -> Dict[str, Any]:
-        return {tag: value for tag, value in self.config.get('tags', {}).items()
-                if tag not in ('clonefrom', 'nofailover', 'noloadbalance', 'nosync') or value}
+        return {
+            tag: value for tag, value in self.config.get('tags', {}).items()
+            if tag not in (
+                'clonefrom',
+                'nofailover',
+                'failover_priority',
+                'noloadbalance',
+                'nosync'
+            ) or value is not False
+        }
 
     @property
     def nofailover(self) -> bool:
-        return bool(self.tags.get('nofailover', False))
+        from_tags = self.tags.get('nofailover', None)
+        if from_tags is not None:
+            # Value of `nofailover` takes precedence over `failover_priority`
+            return bool(from_tags)
+        fp = parse_int(self.tags.get('failover_priority', 1))
+        return fp <= 0 if fp is not None else False
```

Comment:
Avoid using abbreviated variables.
```suggestion
        failover_priority = parse_int(self.tags.get('failover_priority', 1))
        return failover_priority <= 0 if failover_priority is not None else False
```